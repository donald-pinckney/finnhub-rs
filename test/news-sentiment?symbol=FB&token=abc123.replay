{"request":{"url":"https://finnhub.io/api/v1/news-sentiment?symbol=FB&token=abc123","method":"GET","body":null,"headers":{}},"response":{"url":"https://finnhub.io/api/v1/news-sentiment?symbol=FB&token=abc123","status":200,"headers":{"access-control-allow-credentials":"true","access-control-allow-headers":"Origin","access-control-allow-methods":"POST, GET, OPTIONS, PUT, DELETE","access-control-allow-origin":"*","access-control-expose-headers":"Content-Length","alt-svc":"h3-27=\":443\"; ma=86400, h3-28=\":443\"; ma=86400, h3-29=\":443\"; ma=86400","cf-cache-status":"DYNAMIC","cf-ray":"644c3cf73966e1ee-ORD","cf-request-id":"09a3846e840000e1eeeb224000000001","connection":"keep-alive","content-type":"application/json; charset=utf-8","date":"Sat, 24 Apr 2021 03:27:50 GMT","expect-ct":"max-age=604800, report-uri=\"https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct\"","nel":"{\"report_to\":\"cf-nel\",\"max_age\":604800}","report-to":"{\"max_age\":604800,\"group\":\"cf-nel\",\"endpoints\":[{\"url\":\"https:\\/\\/a.nel.cloudflare.com\\/report?s=GnOyZUm2CnTYNQ1X9k0DtrcsM8toZpbGJDmKBbArEJ0Njg8bYFImbU4P4v7fzUhbAkZLavk%2FUm%2BLbrprXs6Wuy5APAGMx%2FRUyplK%2BAu1aMnFS4PaonAD\"}]}","server":"cloudflare","set-cookie":"__cfduid=d9bcbe4c0c9dfb4e588dc2c92dae350ab1619234870; expires=Mon, 24-May-21 03:27:50 GMT; path=/; domain=.finnhub.io; HttpOnly; SameSite=Lax; Secure","transfer-encoding":"chunked","x-ratelimit-limit":"60","x-ratelimit-remaining":"49","x-ratelimit-reset":"1619234930"},"body":"eyJidXp6Ijp7ImFydGljbGVzSW5MYXN0V2VlayI6MTcwLCJidXp6IjowLjg2MDcsIndlZWtseUF2ZXJhZ2UiOjE5Ny41fSwiY29tcGFueU5ld3NTY29yZSI6MC41NjUyLCJzZWN0b3JBdmVyYWdlQnVsbGlzaFBlcmNlbnQiOjAuNjY2Nywic2VjdG9yQXZlcmFnZU5ld3NTY29yZSI6MC41MzU3LCJzZW50aW1lbnQiOnsiYmVhcmlzaFBlcmNlbnQiOjAuMzMzNCwiYnVsbGlzaFBlcmNlbnQiOjAuNjY2Nn0sInN5bWJvbCI6IkZCIn0="},"format_version":3}