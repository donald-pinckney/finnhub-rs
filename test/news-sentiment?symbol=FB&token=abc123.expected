NewsSentiment {
    buzz: Buzz {
        articlesInLastWeek: 170,
        buzz: 0.8607,
        weeklyAverage: 197.5,
    },
    companyNewsScore: 0.5652,
    sectorAverageBullishPercent: 0.6667,
    sectorAverageNewsScore: 0.5357,
    sentiment: Sentiment {
        bearishPercent: 0.3334,
        bullishPercent: 0.6666,
    },
    symbol: "FB",
}
